SURVEY.md

```diff
@@ -32,9 +32,13 @@ We need to figure out pagination, order, pacing, and reasonable length, in addit
     () 200+ employees
 
 ### Location
-    If you live in the US, please choose your state. |v| Dropdown of States |
 
-    If you live outside of the US, please choose your country. |v| Dropdown of Countries |
+    Please choose your country. |v| Dropdown of Countries |
+    
+    Please choose area(?  not sure what the best term is)
```

Comment:
Can we update this to "Which word best describes the area where you live and work?"